examples/news-list/common/aggregates/index.js

```diff
@@ -0,0 +1,15 @@
+export default [
+  {
+    name: 'News',
+    commands: {
+      addNews: (_, { payload: { content, timestamp } }) => ({
+        type: 'NewsAdded',
+        payload: {
+          id: `ID${Math.floor(Math.random() * 1000000000000)}`,
```

Comment:
Math.floor(Math.random() * 1000000000000

Maybe it should be just a GUID?

Reply:
Yes, will be fixed